### PR TITLE
Fix grammatical issues of README.rst + more

### PR DESCRIPTION
**Pull request by ajsharda17**
## Summary

There is a grammatical issue in README.rst, or at least it is hard to read. I have made it more simple to read, as it had confused me.

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [potentially] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by Rapptz** [comment]
Duplicate of https://github.com/Rapptz/discord.py/pull/8082

The README is fine as-is.

**Comment by ajsharda17** [comment]
Sir, it's not a duplicate. This is way different. @Rapptz

**Comment by Rapptz** [comment]
I suggest removing the modifications to the headers then.

**Comment by ajsharda17** [comment]
Alright, thanks

**Comment by ajsharda17** [comment]
Hey @Rapptz, if possible can you check this PR out now? I've updated as per what you said.

File: README.rst [hunk 1]
```diff
@@ -60,7 +60,7 @@ Optional Packages
 
 * `PyNaCl <https://pypi.org/project/PyNaCl/>`__ (for voice support)
 
-Please note that on Linux installing voice you must install the following packages via your favourite package manager (e.g. ``apt``, ``dnf``, etc) before running the above commands:
+Please note that for installing voice on Linux, you must install the following packages via your favourite package manager (e.g. ``apt``, ``dnf``, etc) before running the above commands:
```
Comment (Rapptz) [inline README.rst:63]:
```suggestion
Please note that when installing voice support on Linux, you must install the following packages via your favourite package manager (e.g. ``apt``, ``dnf``, etc) before running the above commands:
```

**Pull request merged**